src/System.Memory/tests/Performance/Perf.Span.Contains.cs

```diff
@@ -0,0 +1,231 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using Microsoft.Xunit.Performance;
+using Xunit;
+
+namespace System.Memory.Tests
+{
+    // Adapted from Perf.Span.IndexOf.cs
+    public class Perf_Span_Contains
+    {
+        private const int InnerCount = 100_000;
+
+        [Benchmark(InnerIterationCount = InnerCount)]
+        [InlineData(1)]
+        [InlineData(10)]
+        [InlineData(100)]
+        [InlineData(1000)]
+        public void SpanContainsChar(int size)
```

Comment:
Add a benchmark using `.IndexOf >= 0`

Reply:
I did so in the comparative benchmarks.

Reply:
Done

Reply:
> I did so in the comparative benchmarks.

In that case, can you share the results for those as well? The results shared here https://github.com/dotnet/corefx/pull/32293#issuecomment-421602959 are comparing vectorized contains to a loop-based one.